packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.dispatchEvent.spec.js

```diff
@@ -0,0 +1,34 @@
+import { createElement } from 'lwc';
+import { extractDataIds } from 'test-utils';
+
+import Container from 'x/container';
+
+function createShadowTree(parentNode) {
+    const elm = createElement('x-container', { is: Container });
+    elm.setAttribute('data-id', 'x-container');
+    parentNode.appendChild(elm);
+    return extractDataIds(elm);
+}
+
+describe('EventTarget.dispatchEvent', () => {
+    let nodes;
+    beforeEach(() => {
+        nodes = createShadowTree(document.body);
+    });
+
+    it('should return true when none of the event handlers cancel', () => {
```

Comment:
Invoking `preventDefault` doesn't cancel an event, the event still propagates normally. But it instructs the browser not to execute the default behavior.

```suggestion
    it('should return true when "Event.preventDefault" is not invoked', () => {
```